lib/model.js

```diff
@@ -2126,50 +2126,12 @@ class Model {
     return this.aggregate(field, 'sum', options);
   }
 
-  /**
-   * Builds a new model instance.
-   *
-   * @param {Object|Array} values An object of key value pairs or an array of such. If an array, the function will return an array of instances.
-   * @param {Object}  [options] Instance build options
-   * @param {boolean} [options.raw=false] If set to true, values will ignore field and virtual setters.
-   * @param {boolean} [options.isNewRecord=true] Is this new record
-   * @param {Array}   [options.include] an array of include options - Used to build prefetched/included model instances. See `set`
-   *
-   * @returns {Model|Array<Model>}
-   */
-  static build(values, options) {
-    if (Array.isArray(values)) {
-      return this.bulkBuild(values, options);
-    }
-
-    return new this(values, options);
-  }
-
-  static bulkBuild(valueSets, options) {
```

Comment:
Are we sure we want to remove this function? After all this has some distinct funtionality.